crates/matrix-sdk/src/room/joined.rs

```diff
@@ -332,9 +333,6 @@ impl Joined {
     /// If the encryption feature is enabled this method will transparently
     /// encrypt the room message if this room is encrypted.
     ///
-    /// **Note**: This method does not support sending custom events, the
```

Comment:
I don't think just removing this will cut it. People won't be familiar with Ruma and won't know how to use Ruma to construct custom events.

Furthermore, I think linking to the raw method has still its merit.

Reply:
Why link to the raw method when it's right below anyways and not needed for 98% of all use cases? Really the only use case I can think of where it makes more sense than just declaring a struct (which can then be used both for sending & handling) is testing server edge cases by sending possibly problematic or intentionally invalid event content.

Reply:
Alright, let's leave the link out.